x-pack/solutions/observability/plugins/slo/public/pages/slo_management/components/slo_management_search_bar.tsx

```diff
@@ -11,17 +11,15 @@ import { EuiComboBox, EuiComboBoxOptionOption, EuiText } from '@elastic/eui';
 import { observabilityAppId } from '@kbn/observability-shared-plugin/common';
 import { useFetchSLOSuggestions } from '../../slo_edit/hooks/use_fetch_suggestions';
 import { useKibana } from '../../../hooks/use_kibana';
+import { SearchState } from './hooks/use_url_search_state';
 
 interface Props {
-  filters: {
-    search: string;
-    tags: string[];
-  };
-  setFilters: Function;
+  state: SearchState;
+  updateFilter: (newState: SearchState) => void;
   onRefresh: () => void;
 }
 
-export function SloManagementSearchBar({ filters, setFilters, onRefresh }: Props) {
+export function SloManagementSearchBar({ state, onRefresh, updateFilter }: Props) {
```

Comment:
I think we could directly use the useUrlSearchState hook in this component so you don't have to provide down state and updateFilter props. Just read and update the url state directly from this component
`const { state, onStateChange } = useUrlSearchState();`